devices/ePaper/Drivers/Ssd168x/Ssd168x.cs

```diff
@@ -18,6 +18,7 @@ namespace Iot.Device.EPaper.Drivers.Ssd168x
     /// </summary>
     public abstract class Ssd168x : IEPaperDisplay
     {
+        private readonly int _maxWaitingTime = 500;
```

Comment:
why not a const?

Reply:
I want to leave the possibility that become a configurable property when instantiating it.